index.bs

```diff
@@ -148,7 +152,7 @@ Usage example {#example}
 The layout shift score is only one signal, which correlates in an approximate
 manner with the user experience of "jumpiness".
 
-Developers should avoid relying on the precise value of the layout shift score,
+Developers have to avoid relying on the precise value of the layout shift score,
```

Comment:
How about "Developers are advised to avoid relying..."

Reply:
Done